### ISSUE TITLE
[BUG] Segmented Control - Hover text color

### ISSUE THREAD

**Issue by henrikvoetmand** [kirbydesign/designsystem#4283]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
Wrong Hover text color on non selected "tabs"

## Describe how to reproduce the bug

1. Navigate to the Segmented Control page on the Cookbook
2. Find the section "Colors"
3. Switch to "dark" theme
4. Mouse over a non selected "tab"

## What was the expected behavior?
Hover text color should be: "kirby-inputs-placeholder-color"

## Add any screenshots
**Cookbook:**
<img width="326" height="89" alt="Image" src="https://github.com/user-attachments/assets/a4bdec2d-ebae-495f-ac24-999f7c70c6f4" />

**Figma:**
<img width="543" height="77" alt="Image" src="https://github.com/user-attachments/assets/60033dc2-7a60-4246-aa35-bfa0aaea520b" />

## Please complete the following information:

- MacOS
- Chrome
- Version 144.0.7559.133 (Officiel version) (arm64)

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.
